### ISSUE TITLE
[BUG] Cannot install the package due to `react-native-reanimated` version conflict

### ISSUE THREAD

**Issue by hpelitebook745G2** [TheWidlarzGroup/react-native-notificated#245]
**Describe the bug**
I cannot install the package in the first place due to `react-native-reanimated` version conflict. My project uses npm and not yarn. 

**To Reproduce**
1. Have a React Native project with versions: 
```
"react-native": "0.72.5",
"react-native-reanimated": "^3.5.4",
```
2. `npm i react-native-notificated`
3. Error:
```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: TestApp@0.0.1
npm ERR! Found: react-native-reanimated@3.5.4
npm ERR! node_modules/react-native-reanimated
npm ERR!   react-native-reanimated@"^3.5.4" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react-native-reanimated@"^2.14.4" from react-native-notificated@0.1.3
npm ERR! node_modules/react-native-notificated
npm ERR!   react-native-notificated@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

**Expected behavior**
To be install the package

**Possible Solution**
To use the version from the root project if detected.

**Comment by PdoubleU** [comment]
Hi @hpelitebook745G2, thanks for bringing it up, we will address this issue and let you know. For now, please try installing the package with `--legacy-peer-deps ` or downgrade the root version to 2.x.x. The library is working with reanimated 3.x.x (I was using it with Expo recently).

**Comment by PdoubleU** [comment]
Hi @hpelitebook745G2, this has been fixed with v0.1.4, please let me know if you have other issues :)
